### PR TITLE
revisions and tests for post-diagnostics project

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Revisions based on Tom's review and lots of additional tests for the post-diagnostics project.

## WHY
This is the last round of work we need before we can ship V1 of the post-diagnostics!

## HOW
An incredible amount of time messing with different view widths to get the table functionality working juuuuust right.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
[(Please provide links to any relevant Notion card(s) relevant to this PR.)
](https://www.notion.so/quill/Post-test-V1-Revisions-02737ddbde544363b864bc72591226d8)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

File: services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_individual_pack.scss [hunk 1]
```diff
@@ -95,18 +96,18 @@
...
-    }
+    // &:after {
+    //   content: '';
+    //   width: 32px;
+    //   background-color: white;
+    //   height: 100%;
+    //   position: absolute;
+    //   right: -33px;
+    //   top: 0px;
+    // }
```
Comment (eadams17) [inline services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_individual_pack.scss]: should we just remove this?

File: services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/__tests__/studentNameOrTooltip.test.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import * as $ from 'jquery'
+
+import StudentNameOrTooltip from '../studentNameOrTooltip'
+
+describe('StudentNameOrTooltip component', () => {
+  it('should render if the name is long enough to warrant a tooltip', () => {
+    const wrapper = mount(<StudentNameOrTooltip
+      name="Gabriel De La Concordia Garcia Gabriel De La Concordia Garcia Gabriel De La Concordia Garcia"
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/__tests__/studentNameOrTooltip.test.jsx:10]: one of my best friends just recommended that I read _Love in the Time of Cholera_  <3

**Review by eadams17 — APPROVED** [review]
🎉

**Review by cissyxyu — APPROVED** [review]

**Review by anathomical — APPROVED** [review]
So pumped about this.

**Pull request merged**